src/errors.rs

```diff
@@ -53,6 +53,9 @@ pub enum FlokiError {
     #[fail(display = "Failed to check existance of image '{}': {}", image, error)]
     FailedToCheckForImage { image: String, error: io::Error },
 
+    #[fail(display = "Failed to find the specified key")]
```

Comment:
We can probably give some more specific diagnostics here (failed to find key 'foo.bar.baz' in file 'config.yaml').

Reply:
ok done